.github/workflows/release.yml

```diff
@@ -27,10 +27,10 @@ jobs:
           cache: 'pnpm'
 
       - name: Install Dependencies
-        run: pnpm install
+        run: pnpm install --frozen-lockfile
```

Comment:
Should be frozen by default in CI, but i guess better safe than sorry.

Reply:
Something must have gone wrong then, because the lockfile was out of date.